apps/ledger-live-desktop/tests/component/modal.component.ts

```diff
@@ -117,4 +118,15 @@ export class Modal extends Component {
   async waitForConfirmationScreenToBeDisplayed() {
     await this.confirmText.waitFor({ state: "visible" });
   }
+
+  async scrollUntilOptionIsDisplayed(dropdown: Locator, element: Locator) {
+    let isVisible = await element.isVisible();
+
+    while (!isVisible) {
+      await dropdown.evaluate(node => {
+        node.scrollBy(0, 100);
```

Comment:
I don't think so because with 100 we don't skip any tokens from the list.

https://github.com/user-attachments/assets/3afbbe0b-c6cf-4212-9a2c-aafb70862fb8